cfunits/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: cfunits
+  version: {{environ['CBUILD_CFUNITS_VERSION']}}
+
+source:
+  git_url: https://bekozi@bitbucket.org/cfpython/cfunits-python.git
+  git_tag: {{environ['CBUILD_CFUNITS_TAG']}}
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
I see you already added `--record record.txt`.  Thanks!  All modules that use `setuptools` we build with:

`--single-version-externally-managed --record record.txt`

That makes our life much easier in the long run.